client/ContributionsListView.js

```diff
@@ -56,6 +55,23 @@ const styles = theme => ({
 
 class MainPageTB extends Component {
 
+    constructor(props) {
+        super(props);
+        this.state = {
+            publishedList: null,
+        };
+    }
+
+    componentDidMount() {
+        if (this.props.publishedList) {
+            fb.base.syncDoc(this.props.publishedList.ref.path, {
```

Comment:
move `syncDoc` call to `App.js` and pass down list via props to avoid duplicate firebase calls